app/jobs/address_proofing_job.rb

```diff
@@ -10,8 +10,8 @@ def perform(result_id:, encrypted_arguments:, callback_url:, trace_id:)
     Idv::Proofer.address_job_class.handle(
       event: {
         applicant_pii: decrypted_args[:applicant_pii],
-        callback_url: :callback_url,
-        trace_id: :trace_id,
+        callback_url: callback_url,
+        trace_id: trace_id,
```

Comment:
we can probably drop these soon right?

Reply:
yeah, we can and should, I think it's safe to pass nil by default until we remove it from the functions themselves. We _could_ keep trace_id if we wanted since it probably does exist, but I don't know how useful it would be.